example/src/main/java/com/genius/groupie/example/item/CardItem.java

```diff
@@ -3,13 +3,13 @@
 import android.support.annotation.ColorRes;
 
 import com.genius.groupie.Item;
-import com.genius.groupie.example.MainActivity;
 import com.genius.groupie.example.databinding.ItemCardBinding;
 
 import java.util.HashMap;
 import java.util.Map;
 
-import static com.genius.groupie.example.MainActivity.*;
```

Comment:
It just messes up git blame. :)  I hope to get a style check going one of these days!